src/anniversary/cri/libadxe/adx_fcch.c

```diff
@@ -0,0 +1,7 @@
+#include "cri/private/libadxe/adx_fcch.h"
+#include "common.h"
+#include "sdk/eekernel.h"
+
+void ADXF_Ocbi(Sint8 *buf, Sint32 bsize) {
+    InvalidDCache(buf, &buf[bsize] - 1);
```

Comment:
This can be simplified a bit
```suggestion
    InvalidDCache(buf, buf+ bsize - 1);
```